proposals/NNNN-font-styles.md

```diff
@@ -0,0 +1,111 @@
+# Font Styles
+
+* Proposal: [NNNN](NNNN-font-styles.md)
+* SDL Proposal: [To be filled later](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/)
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / Web / RPC ]
+
+## Introduction
+
+This feature would let apps bold, underline or italicize text within Alert, Show or Perform Interaction RPCS.
+
+## Motivation
+
+As an app partner, I want to be able to customize text on main layouts and popups so that I can have a visually pleasing and easy to use UI.
+
+## Proposed solution
+
+
+When an app wants a more visually appealing UI experience,  it can add a font style cutomization onto each line of text if desired.  If no font style is provided, then it is assumed that the font is normal.  
```

Comment:
```suggestion
When an app wants a more visually appealing UI experience,  it can add a font style customization onto each line of text if desired.  If no font style is provided, then it is assumed that the font is normal.  
```